pkgs/development/libraries/libblocksruntime/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, clang }:
+
+stdenv.mkDerivation {
+  name = "blocksruntime";
+
+  src = fetchFromGitHub {
+    owner = "mackyle";
+    repo = "blocksruntime";
+    rev = "b5c5274daf1e0e46ecc9ad8f6f69889bce0a0a5d";
+    sha256 = "0ic4lagagkylcvwgf10mg0s1i57h4i25ds2fzvms22xj4zwzk1sd";
+  };
+
+  buildInputs = [ clang ];
+
+  configurePhase = ''
+    export CC=clang
```

Comment:
I'd actually use the `clangStdenv` here, since that'll get you proper llvm-flavored libraries like libc++ and so on, and will be more standard I think.